docs/developers/docker.rst

```diff
@@ -61,6 +61,29 @@ data, run:
 
     docker-compose -f docker-compose-dev.yml rm -f
 
+Interactive sessions
+--------------------
+
+For interactive sessions inside the container simply attach a shell to the
+running instance:
+
+.. code-block:: shell
+
+    docker exec -it invenio_web_1 bash
+
+.. note::
+    Some tools (mostly ncurses-based ones) require the ability to detect the
+    used terminal. To enable this, set the `TERM` environment variable to the
+    correct value as a first command inside the container, e.g. by running:
+
+    .. code-block:: shell
+
+        export TERM=xterm
```

Comment:
I wonder what prompted you to make this addition. Having to redefine TERM implies issues with the way the environment is set up the majority of times.